cmd/exec/main.go

```diff
@@ -651,113 +635,65 @@ EXAMPLE: direktiv-sync event greeting.yaml
 	},
 }
 
-var eventfilterCmd = &cobra.Command{
-	Use:   "eventfilter operation EVENTFILTER_PATH",
-	Short: "Remotely manage direktiv eventfilter",
-	Long: `Remotely manage direktiv eventfilters. This process can delete, create and update eventfilter.
-EXAMPLE: direktiv-sync event create -s script.txt -f filtername
-	     direktiv-sync event delete filternanme
-		 direktiv-sync event update -s script.txt -f filtername
-`,
+var eventsCmd = &cobra.Command{
+	Use:   "events",
+	Short: "Event-related commands.",
 }
 
-var createCmd = &cobra.Command{
-	Use:   "create SCRIPT_PATH",
-	Short: "Remotely manage direktiv eventfilter",
-	Long: `Remotely manage direktiv eventfilters. This process can delete, create and update eventfilter.
-EXAMPLE: direktiv-sync event create -s script.txt -f filtername
-	     direktiv-sync event delete filternanme
-		 direktiv-sync event update -s script.txt -f filtername
-`,
-
-	Args: cobra.ExactArgs(1),
+var setFilterCmd = &cobra.Command{
+	Use:   "set-filter NAME SCRIPT",
```

Comment:
NAME became a required argument instead of a flag. There are many schools of thought about how CLIs should be designed, but my opinion is that *usually* things that are required should be arguments and things that are optional should be flags.